.babelrc

```diff
@@ -40,9 +38,8 @@
         "transform-es2015-parameters",
         "transform-es2015-destructuring",
         "transform-es2015-block-scoping",
-        "transform-es2015-typeof-symbol",
-        "transform-object-assign"
+        "transform-es2015-typeof-symbol"
       ]
     }
   }
-}
+}
```

Comment:
yep, since i opted to stick with babel rather than buble due to the fact that we already use `babelify` with `watchify` for development. i couldn't find an equivalent hot-reload for buble.